tests/bdd/session/session_test.go

```diff
@@ -0,0 +1,190 @@
+package session_test
+
+import (
+	. "github.com/onsi/ginkgo"
+)
+
+// ************************************************************************************************************
+// Milestone: Sessions
+//
+// Tentative Timeline (4-6 weeks)
+//
+// Unanswered Questions?
+// - Where is the seed data coming from? 99% sure -> *WORLD STATE*
+// - How is the state of the session nodes maintained?
+// - Is service request validation ddos safe?
+// - Forking behavior
+// ************************************************************************************************************
+
+var _ = Describe("Session", func() {
+
+	Describe("Session Creation \\ Computing", func() {
+
+		Context("Invalid Seed Data", func() {
+			// n-1 Block Hash (probably world state), blockchains requested [list] (probably  world state), node [list] (probably world state))
+
+			Context("Parameters are missing or null", func() {
+
+				It("should error", func() {
+					// Code goes here ...
+				})
+			})
+
+			Context("Devid1 is incorrect...", func() {
+
+				Context("Devid is incorrect format", func() {
+
+					It("should error", func() {
```

Comment:
This is a good place (as elsewhere in this PR) to say what the error should be. "It should return an 'invalid developer id' error" or something similar. And somewhere we should answer: to whom is that error returned, and how?

Reply:
I agree with Irving on this one, as much as behaviours inform interfaces, they also inform errors and how we design them.

Reply:
Okay, since the sprint should start today, I think I should go ahead and start developing based on the 'happy paths' we agree on. Then we can meet to discuss this further